packages/ui/src/lib/NativeBase/Navbar/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { Image, Box, HamburgerIcon, HStack, Button, Square } from "native-base";
+import PropTypes, { InferProps } from "prop-types";
+import React, { useReducer } from "react";
+import MenuItem, { menuItemPropTypes } from "./menuItem";
+
+const navbarPropTypes = {
+  items: PropTypes.arrayOf(PropTypes.shape(menuItemPropTypes)).isRequired,
+  logo: PropTypes.shape({
+    source: PropTypes.string.isRequired,
+    url: PropTypes.string.isRequired,
+  }).isRequired,
+};
+
+type NavbarProps = InferProps<typeof navbarPropTypes>;
+
+const style = {
+  navbar: {
+    height: ["50px", "50px", "70px"],
+    px: ["15px", "15px", "30px"],
+  },
+  logo: {
+    width: ["101px", "101px", "144px"],
+    height: ["35px", "35px", "50px"],
+  },
+};
+
+const display = {
+  flexOnDesktop: ["none", "none", "flex"],
+  flexOnMobile: ["flex", "flex", "none"],
+};
+
+export default function Navbar({ items, logo }: NavbarProps) {
```

Comment:
Actually, I also spent much time thinking about this. In my opinion, Navbar should only be a stupid/presentational component. If data is hardcoded inside, it will be hard to adjust the items dynamically.

For example, if a user is not logged in, we may want to show the "login/signup" item on Navbar. Otherwise, we may want to show the "logout" item, so I think it's important to let Navbar's item be modifiable from outside. What’s your take on this?

Reply:
That's a good question, let me think about it, but can't logo be hard-coded?

Reply:
Agree, for the logo, I think we can hardcode it.

Reply:
To pass text as props:
1. We can decouple _t with the component
2. Wherever we use navbar, we have to pull in the menu items separately

Reply:
Let's clarify, do you mean we add this to our design system? A responsive navbar component that's generic?

Reply:
> Wherever we use navbar, we have to pull in the menu items separately

Right, I think we can only use Navbar once in the Layout.js, so that we don't have to pull in the menu items in many files.

> Let's clarify, do you mean we add this to our design system? A responsive navbar component that's generic?

Yes, so I think the Navbar component should not contain data.

Reply:
We can create another component with hard-coded data and use this navbar, this way, we can reduce code in the controller.

Reply:
I think we can create 2 ui libs, one for generic components that doesn't have hard-coded data, the other uses the generic one with hard-coded data.

Reply:
I concur. I think we can put those smart components in pokemon and keep presentational and stupid components in @utofun/ui.